packages/stark-core/src/modules/settings/entities/settings.entity.ts

```diff
@@ -5,5 +5,5 @@ export class StarkSettings {
 	/**
 	 * the user's preferred language
 	 */
-	public preferredLanguage: string;
+	public preferredLanguage?: string;
```

Comment:
Not always, at 
`packages/stark-core/src/modules/settings/reducers/settings.reducer.ts(12:0)`
 it will be undefined. 😕 Perhaps I could assign it a default value? (`""`)